src/components/Button/Button.style.css

```diff
@@ -0,0 +1,62 @@
+.button {
+  width: 150px;
+  height: 44px;
+  text-align: center;
+  font-weight: bold;
+  font-size: 18px;
+  border-radius: 4px;
+  border-width: thin;
+  background-color: #f2f1e7;
+  font-family: "Orqquidea-Sans_(DEMO)";
+  flex-shrink: 1;
+}
+.button--red {
+  border-color: red;
+  color: red;
+}
+.button--regular {
+  border-color: black;
+  color: black;
+}
+.button--white {
+  border-width: 0px;
+  color: black;
+}
+.button--seeMore {
```

Comment:
Choose one naming convention or the other. I recommend kebab-case/snake-case, since we are using BEM